changelog.md

```diff
@@ -5,7 +5,8 @@
 ### Version 4.0.9
 
 `api-scanner`: Remove the `@JvmStatic` annotation from the generated output.
-
+`cordformation`: Removed default dependency on jolokia, to provide HTTP JMX monitoring, explicitly depend on jolokia.
```

Comment:
Oops, we have a formatting issue here. The markdown should be an itemised list, so this should be:
```markdown
* `api-scanner`: ...
* `cordformation`: ...
* `noderunner`: ...
```
Unfortunately I forgot to start some of the lines with `*`. Could you add them back in please?